database/migration/V20161226_0927__createAttendees.sql

```diff
@@ -115,3 +115,16 @@ CREATE TABLE `attendee_project_interests` (
 		ON DELETE NO ACTION
 		ON UPDATE NO ACTION
 );
+
+CREATE TABLE `attendee_rsvps` (
+	`id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+	`attendee_id` INT UNSIGNED NOT NULL,
+	`attendee_attendance` ENUM('YES', 'NO', 'YES_TO_CREATE') NOT NULL,
```

Comment:
In order to get this out of your local database's flyway version control, you'll have to delete the corresponding entry in the `schema_version` table (otherwise it'll complain when you run it). Let me know if you need help with that.